src/ActivitylogServiceProvider.php

```diff
@@ -26,6 +26,14 @@ public function boot()
                 __DIR__.'/../migrations/create_activity_log_table.php.stub' => database_path("/migrations/{$timestamp}_create_activity_log_table.php"),
             ], 'migrations');
         }
+
+        if (!class_exists('UpdateActivityLogTable')) {
+            $timestamp = date('Y_m_d_His', time());
```

Comment:
I have no idea if it's the best way - but right now both migrations could get the same timestamp. How about doing `time() + 1` ?